tectonic/msvc-include/getopt.h

```diff
@@ -0,0 +1,655 @@
+// Taken from https://github.com/skandhurkat/Getopt-for-Visual-Studio/blob/master/getopt.h
+
```

Comment:
We do not use `getopt` or `getopt_long` anymore. Could you instead remove the `#include <getopt.h>` from `internals.h`? The include is useless now.